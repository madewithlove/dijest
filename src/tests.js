export {default as component} from './tests/component';
export {reducerTestFactory, default as reducer} from './tests/reducer';
export {default as saga} from './tests/saga';
