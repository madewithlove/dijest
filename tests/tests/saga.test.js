import React from 'react';
import {saga} from '../../dist/tests';

const dummySaga = function*(action) {
    yield action;
};

saga('can test saga', dummySaga({foo: 'bar'}));
saga('can test saga with result', dummySaga({foo: 'bar'}), [{action: {foo: 'bar'}}]);