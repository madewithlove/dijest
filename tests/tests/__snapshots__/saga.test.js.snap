exports[`test can test saga 1`] = `
Array [
  Object {
    "done": false,
    "value": Object {
      "foo": "bar"
    }
  }
]
`;
